### DISCUSSION TITLE
unfortunately I have to use 0.31

### DISCUSSION THREAD

**Discussion by daveandbambifan1234** [JordanSantiagoYT/FNF-JS-Engine#539] [General]
@JordanSantiagoYT I have to use the latest version without these 0.7.X version of the psych engine note style cuz I have 0.6.3 but I'm going to try resolving this issue because the game kept crashing in every song
but in future mods I will use 0.33.0 or more greater versions of the engine.
IDK why I made this comment💀

**Comment by TheStinkern** [comment]
what

**Comment by moxie-coder** [comment]
> @JordanSantiagoYT I have to use the latest version without these 0.7.X version of the psych engine note style cuz I have 0.6.3 but I'm going to try resolving this issue because the game kept crashing in every song but in future mods I will use 0.33.0 or more greater versions of the engine. IDK why I made this comment💀

then use a different engine or something, idk why the new note skins are such a big deal with you when all they do is make things easier and less complicated

**Reply by imavie** [reply]
you should ban him from the repo if that option even exists

**Reply by moxie-coder** [reply]
> you should ban him from the repo if that option even exists

I can’t but he can
